### ISSUE TITLE
Proof Array Question

### ISSUE THREAD

**Issue by kayaelle** [1EdTech/openbadges-specification#473]
Verifiable Credentials allow for multiple signatures but does not require proof to be an array. Why does the schema require this?

**Comment by mgylling** [comment]
As per [Appendix A.1](https://imsglobal.org/spec/ob/v3p0/#json) in the draft, collections must be expressed as arrays. This follows from a restriction (intended to result in simplification for consumers) that was minted in CLR 1.0. While the CLR group has expressed support for this restriction, the OB group may not be aware of it. Needs working group discussion.

**Comment by martyr280** [comment]
@mgylling would like this on the agenda for 10/20 workgroup as well.

**Comment by xaviaracil** [comment]
Solved in working group meeting on 10/22: Add compaction support for arrays .

**Comment by kayaelle** [comment]
Re-opening this issue because the schema still seems to require proof to be an array. 

Example here: https://www.jsonschemavalidator.net/s/keh9hfs3

**Comment by xaviaracil** [comment]
Which JSON schema are you using? There's a section in the spec about JSON-LD with dedicated JSON schemas that supports JSON-LD features, such as array compaction (https://www.imsglobal.org/spec/ob/v3p0#schemas)

**Comment by kayaelle** [comment]
Oh I see - there's the schema at the top of the spec: https://purl.imsglobal.org/spec/ob/v3p0/schema/**json**/ob_v3p0_achievementcredential_schema.json

And then there's a second schema referenced at the bottom of the spec: https://www.imsglobal.org/spec/ob/v3p0#schemas
https://purl.imsglobal.org/spec/ob/v3p0/schema/**json-ld**/ob_v3p0_achievementcredential_schema.json


Can we just have one schema that will support all JSON-LD features since OBv3 is in JSON-LD? That would greatly alleviate confusion and help with interoperability (and save a lot of time for people who don't know they should look for a second schema).

**Comment by xaviaracil** [comment]
Agreed to use the JSON-LD featured JSON Schema only at the August 28th meeting.
